core/controlplane/config/templates/cloud-config-controller

```diff
@@ -394,6 +408,16 @@ ssh_authorized_keys:
         WantedBy=install-kube-system.service
 {{end}}
 write_files:
+{{- if .Controller.CustomWriteFiles}}
```

Comment:
I prefer `customFiles`/`CustomFiles` rather than `CustomWriteFiles`. I believe it is named after `write_files` in cloud-config but I guess that's an implementation detail. Although it isn't followed everywhere but I'd like to hide implementation details whenever possible.